watsonsdk/AuthConfiguration.h

```diff
@@ -20,12 +20,14 @@
 
 @property NSString* basicAuthUsername;
 @property NSString* basicAuthPassword;
+@property (nonatomic) BOOL xWatsonLearningOptOut;
 
 @property (readonly) NSString *token;
 @property (copy, nonatomic) void (^tokenGenerator) (void (^tokenHandler)(NSString *token));
 
 - (void) invalidateToken;
 - (void) requestToken: (void(^)(AuthConfiguration *config)) completionHandler;
 - (NSDictionary*) createRequestHeaders;
+- (NSDictionary*) createRequestHeadersWithXWatsonLearningOptOut;
```

Comment:
It is a good idea to separate them, changing configuration is good too, either way is OK.